src/js/features/live-viewers.js

```diff
@@ -0,0 +1,66 @@
+var debug = require('../helpers/debug');
+
+function checkBind() {
+    var isBound = false;
+    if ($('.live-count').length) {
+        $.each($('.live-count').data('events'), function(i) {
+            if (i === 'DOMSubtreeModified') {
+                isBound = true;
+                return;
+            }
+        });
+    }
+    return isBound;
+}
+
+module.exports = function() {
+    var $div = $('div.app-main');
+    var viewerSpan = '<span id="viewerSpan">"Loading Viewers...</span>';
+    $('div.player-livestatus').append(viewerSpan);
+    $('#viewerSpan').text(function() {
+        return $('.live-count').text() + 'Viewers';
+    });
+
+    if (checkBind() === true) {
+        $('.live-count').unbind('DOMSubtreeModified');
+    }
+
+    debug.log('Creating viewer count span');
+    $('.live-count').bind('DOMSubtreeModified', function() {
```

Comment:
`DOMSubtreeModified` is deprecated and should be replaced with mutation observers. You should also be able to drop the `unbind` since garbage collection should take care of the mutation observer when the watched element is removed.